docs/deploy_customization.md

```diff
@@ -0,0 +1,104 @@
+
+# Azure AI Foundry Starter Template: Deployment customization
+
+This document describes how to customize the deployment of the Azure AI Foundry Starter Template. Once you follow the steps here, you can run `azd up` as described in the [Deploying](./README.md#deploying) steps.
+
+* [Disabling resources](#disabling-resources)
+* [Customizing resource names](#customizing-resource-names)
+* [Customizing model deployments](#customizing-model-deployments)
+
+## Disabling resources
+
+* To disable AI Search, run `azd env set USE_SEARCH_SERVICE false`
+* To disable Application Insights, run `azd env set USE_APPLICATION_INSIGHTS false`
+* To disable Container Registry, run `azd env set USE_CONTAINER_REGISTRY false`
+
+Then run `azd up` to deploy the remaining resources.
```

Comment:
Consider rewording

After that `azd up` will not deploy the resources you have disabled.